src/Cli/func/ExtensionBundle/ExtensionBundleHelper.cs

```diff
@@ -49,13 +55,17 @@ public static string GetBundleDownloadPath(string bundleId)
         public static async Task GetExtensionBundle()
         {
             var extensionBundleManager = GetExtensionBundleManager();
+
             try
             {
-                using (var httpClient = new HttpClient())
-                {
-                    httpClient.Timeout = TimeSpan.FromMinutes(10);
-                    await extensionBundleManager.GetExtensionBundlePath(httpClient);
-                }
+                using var httpClient = new HttpClient { Timeout = _httpTimeout };
```

Comment:
The `GetExtensionBundle` method is called from `StartHostAction.RunAsync`. I am wondering that a 10 minute HTTP timeout might be too long. If the request hangs, users running `func host start` command could be left waiting without feedback and may think the CLI is unresponsive. Since we already have retries in place, maybe we could consider reducing the timeout to fail faster and improve the user experience?